docs/Configuration/Dialplan/Pattern-Matching.md

```diff
@@ -3,331 +3,255 @@ title: Pattern Matching
+When Alice dials `6421`, Asterisk searches through its list of sorted extensions and uses the first matching extension. In this case `_64NX` is found.
 
-```
-exten => _6XX1,1,SayAlpha(A)
-exten => _64XX,1,SayAlpha(B)
-exten => _640X,1,SayAlpha(C)
-exten => _6.,1,SayAlpha(D)
-exten => _64NX,1,SayAlpha(E)
-exten => _6[45]NX,1,SayAlpha(F)
-exten => _6[34]NX,1,SayAlpha(G)
+To verify that Asterisk actually does sort the extensions in the manner that we've shown, copy the example dialplan to `extensions.conf` and type `dialplan reload` at the Asterisk CLI. The dialplan then becomes:
 
 ```
-
-Reload the dialplan, and then type **dialplan show 6421@users** at the Asterisk CLI. Asterisk will show you all extensions that match in the **[users]** context. If you were to dial extension **6421** in the **[users]** context the first found extension will execute.
-
-```
-server*CLI> dialplan show 6421@users
-[ Context 'users' created by 'pbxp_config' ]
+server*CLI> dialplan show users
+[ Context 'users' created by 'pbx_config' ]
+ '_640X' => 1. SayAlpha(C) [pbx_config]
  '_64NX' => 1. SayAlpha(E) [pbx_config]
  '_64XX' => 1. SayAlpha(B) [pbx_config]
  '_6[34]NX' => 1. SayAlpha(G) [pbx_config]
  '_6[45]NX' => 1. SayAlpha(F) [pbx_config]
  '_6XX1' => 1. SayAlpha(A) [pbx_config]
  '_6.' => 1. SayAlpha(D) [pbx_config]
 
--= 6 extensions (6 priorities) in 1 context. =-
-
+-= 7 extensions (7 priorities) in 1 context. =-
 ```
+
+To show all extensions that match in the `[users]` context, type `dialplan show 6421@users` at the Asterisk CLI:
+
 ```
-server*CLI> dialplan show users
-[ Context 'users' created by 'pbx_config' ]
- '_640X' => 1. SayAlpha(C) [pbx_config]
+server*CLI> dialplan show 6421@users
+[ Context 'users' created by 'pbxp_config' ]
  '_64NX' => 1. SayAlpha(E) [pbx_config]
  '_64XX' => 1. SayAlpha(B) [pbx_config]
  '_6[34]NX' => 1. SayAlpha(G) [pbx_config]
  '_6[45]NX' => 1. SayAlpha(F) [pbx_config]
  '_6XX1' => 1. SayAlpha(A) [pbx_config]
  '_6.' => 1. SayAlpha(D) [pbx_config]
 
--= 7 extensions (7 priorities) in 1 context. =-
+-= 6 extensions (6 priorities) in 1 context. =-
 
 ```
 
-You can dial extension **6421** to try it out on your own.
+If you were to dial extension **6421** in the **[users]** context the first found extension would execute (*i.e.* `_64NX`).
 
+You can dial extension **6421** to try it out on your own.
 
 
+Auto-Fallthrough
+----------------
 
 !!! warning Be Careful with Pattern Matching
-    Please be aware that because of the way auto-fallthrough works, if Asterisk can't find the next priority number for the current extension or pattern match, it will also look for that same priority in a less specific pattern match. Consider the following example:
+    Please be aware that because of the way auto-fallthrough works, if Asterisk can't find the next priority number for the current extension or pattern match, it will also look for that next priority in a less specific pattern match.
```

Comment:
If I write it as *"it will look for that next priority in a less specific pattern match"*, does it still represents the correct behaviour? I thought it's clearer.

Reply:
I would leave it as "same"